_health-care/_js/components/ProgressButton.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A component for the continue button to navigate through panels of questions.
+ *
+ * Required props
+ * `onButtonClick`: function that changes the path to the next panel or submit.
+ * `buttonText`: String. Stores the value for the button text.
+ * `buttonClass`: String. Stores the value for the button class(es).
+ */
+
+class ProgressButton extends React.Component {
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  render() {
+    return (
+      <div className="row progress-buttons">
+        <div className="small-8 columns">
```

Comment:
Actually it's not that simple because 2 buttons won't be inline since they are both individually wrapped with a `row` class. We could remove these Foundation classes, since we eventually want to get away from Foundation anyway, and either add usa-grid classes to style them inline or just manually do it by adding a float to the buttons. If we use the usa-grid classes, it'll be the same problem as Foundation where we'll want to extract the `row` equivalent class out of this component so they appear in the same row.